convert2rhel/unit_tests/utils_test.py

```diff
@@ -173,7 +173,7 @@ def test_download_pkg_assertion_error(self):
     @unit_tests.mock(system_info, "releasever", "7Server")
     @unit_tests.mock(system_info, "version", namedtuple("Version", ["major", "minor"])(7, 0))
     @unit_tests.mock(utils, "run_cmd_in_pty", RunSubprocessMocked(ret_code=1))
-    @unit_tests.mock(os, "environ", {"CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK": "1"})
+    @unit_tests.mock(os, "environ", {"CONVERT2RHEL_INCOMPLETE_ROLLBACK": "1"})
```

Comment:
To maintain complete coverage, you will need to test when `CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK` is set and also test when `CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK` is set.  (For completeness, I would also test when they are both set).

You can probably use `pytest.mark.parametrize` for that.  I can help you with that if you need it.

Reply:
Oh, @abadger, I believe you meant `CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK` and `CONVERT2RHEL_INCOMPLETE_ROLLBACK`, right?

You pasted `CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK` twice. 

+1 to that, believe that settings both of them in a test is a good thing to have.

Reply:
Thanks guys, for the input on this! I am going to be pushing the updated unit test for this today!